### PR TITLE
Solved multiple issues

### PR DESCRIPTION
**Pull request by rinoma**
update package.json rlates #1
update new file file structure #2
create gitignore #3
create file structure #4
setup server #5
create db #6
create routes #8
create basic layout #9
create some basic partials like footer #10
create app.js #11
create css and favicon #12

### CODE REVIEW

**Review by tyeung2017 — APPROVED** [review]
basic layout of our app

**Comment by rinoma** [comment]
what a fine job

**Comment by tyeung2017** [comment]
hey mario im an idiot

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
-# facnrecipev4
\ No newline at end of file
+# Facn Recipe v4.0
+
+
+# FACN Recipe 2.0
```
Comment (Karyum) [inline README.md]: i thought this was v4.0

File: README.md [hunk 2]
```diff
@@ -1 +1,69 @@
...
+The ultimate FACN Recipe app.
+A simple recipe application for members of Founders & Coders community in Nazareth.
+
+### To get started:
+Click [here]() to use FACN Recipe 2.0.
```
Comment (Karyum) [inline README.md]: also here v4.0

File: database/dbConnection.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+const herokuDB = {
+  host: process.env.HEROKU_HOST,
+  user: process.env.HEROKU_USER,
+  password: process.env.HEROKU_PW,
+  database: process.env.HEROKU_DB,
+  ssl: true,
+};
+
+const localDB = {
```
Comment (Karyum) [inline database/dbConnection.js]: think about this for a second are you sure you want this file to be like this? and are you using a heroku remote database cause if yes you dont need this const localDB

File: src/controllers/index.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+// //Routes
```
Comment (Karyum) [inline src/controllers/index.js:1]: commented out code

File: package.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    "type": "git",
+    "url": "git+https://github.com/facn2/facnrecipev4.git"
+  },
+  "author": "Leo and Stefano",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/facn2/facnrecipev4/issues"
+  },
+  "homepage": "https://github.com/facn2/facnrecipev4#readme",
+  "dependencies": {
```
Comment (Karyum) [inline package.json:21]: do you need all the dependencies here?

**Review by Karyum — CHANGES_REQUESTED** [review]
You should never. Ever do this.
1 issue is 1 pull request.
Never make a pull request in which you reference multiple issues.
I'm assigning both of you and waiting for a thumbs up so i can see that you have read this.

File: src/controllers/index.js (hunk 4 above)
Comment (rinoma) [reply src/controllers/index.js:1]: we need this commented out code

**Comment by Karyum** [comment]
#1 #2 #3 #4 should be 1 PR
#5 #8 #11 1 PR
#6 1 PR
#9 #10 1 PR
#12 1 PR

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1,107 @@
+node_modules/
+config.env
+
```
Comment (philawsophizing) [inline .gitignore]: Do you know what all these things are? Why ignore things that don't exist..?

File: package.json (hunk 5 above)
Comment (rinoma) [reply package.json:21]: maybe

Comment (philawsophizing) [reply package.json:21]: As Mario said, require dependencies and dev dependencies as needed. That way you can link all the things relevant to an issue at the same time (just files changed, packages needed etc for that issue)

**Review by philawsophizing — COMMENTED** [review]
Better to bombard us with a smaller pull request for each issue you resolve than change 16! files with 5751! lines and most of your outstanding issues.

File: .gitignore (hunk 6 above)
Comment (rinoma) [reply .gitignore]: Cleaned it, the rest is standerd for everyproject. They don't exist because they are ignored.

File: package.json (hunk 5 above)
Comment (rinoma) [reply package.json:21]: the ones that are there should be the ones that we are going to use, we plan ahead.

Comment (philawsophizing) [reply package.json:21]: And two commits for that pull request? 'Solved multiple issues' is not a good commit message. One commit for each file changed or at least each issue referenced is a good rule of thumb.

Comment (Karyum) [reply package.json:21]: @rinoma you plan ahead but never implement ahead

**Comment by Karyum** [comment]
@rinoma @tyeung2017 next pull reuqest that is this big **will not be merged**
# 1 Issue 1 Pull Request

**Review by Karyum — APPROVED** [review]

**Pull request merged**